dotcom-rendering/src/web/layouts/StandAlonePage.tsx

```diff
@@ -0,0 +1,203 @@
+import { css, Global } from '@emotion/react';
+import {
+	brandAlt,
+	brandBackground,
+	brandBorder,
+	brandLine,
+	focusHalo,
+	neutral,
+	palette,
+} from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import type { ReactNode } from 'react';
+import { StrictMode } from 'react';
+import type { NavType } from '../../model/extract-nav';
+import type { DCRNewslettersPageType } from '../../types/newslettersPage';
+import { AlreadyVisited } from '../components//AlreadyVisited.importable';
+import { FocusStyles } from '../components//FocusStyles.importable';
+import { Footer } from '../components/Footer';
+import { Header } from '../components/Header';
+import { HeaderAdSlot } from '../components/HeaderAdSlot';
+import { Island } from '../components/Island';
+import { Nav } from '../components/Nav/Nav';
+import { ReaderRevenueDev } from '../components/ReaderRevenueDev.importable';
+import { Section } from '../components/Section';
+import { SkipTo } from '../components/SkipTo';
+import { SubNav } from '../components/SubNav.importable';
+import { Stuck } from '../layouts/lib/stickiness';
+import { decideFormat } from '../lib/decideFormat';
+
+type Props = {
+	model: DCRNewslettersPageType;
+	nav: NavType;
+	children: ReactNode;
+};
+
+/**
+ * @description
+ * Article is a high level wrapper for stand alone pages on Dotcom. Sets strict mode and some globals
+ *
+ * */
+export const StandAlonePage = ({ children, model, nav }: Props) => {
```

Comment:
I'd love to chat a bit more about what the standalone page definition is (I think it comes from Frontend?) and if it's something we'd want to keep in DCR, or if we'd favour simply having an EmailNewsletterPage template/component for example